CHANGELOG

```diff
@@ -15,6 +15,7 @@ Unreleased
 - add config option `timeout-interval` to terminate the service in case it hangs
 - add support to Fluentbit log forwarding
 - change OIDC configurations to support any kind of OIDC provider
+- change package provider to PyPI
```

Comment:
Technically, the charm was using PyPI. But not The PyPI.org, it was Omnivector's private PyPI. What do you think about this:

```suggestion
- change package provider to PyPI.org
```

Reply:
solved in [a769edf](https://github.com/omnivector-solutions/charm-license-manager-agent/pull/40/commits/a769edf242daef222cd2133ccc49a9aaa9224149)